slsDetectorSoftware/src/Detector.cpp

```diff
@@ -768,7 +768,7 @@ void Detector::setRxAddGapPixels(bool enable) {
 Result<bool> Detector::getParallelMode(Positions pos) const {
     auto res = pimpl->Parallel(&slsDetector::setReadOutFlags, pos,
                                defs::GET_READOUT_FLAGS);
-    Result<bool> booleanRes;
+    Result<bool> booleanRes(res.size());
     for (unsigned int i = 0; i < res.size(); ++i) {
         booleanRes[i] = (res[i] & defs::PARALLEL) ? true : false;
```

Comment:
This code should be equivalent. Since both versions rely on implicit conversion. 
booleanRes[i] = res[i] & defs::PARALLEL;

Reply:
done